crates/cxx-qt-lib/src/gui/qpainter.rs

```diff
@@ -158,6 +163,10 @@ mod ffi {
         #[rust_name = "set_opacity"]
         fn setOpacity(self: Pin<&mut QPainter>, opacity: f64);
 
+        /// Draws the outline (strokes) the path path with the pen specified by pen
+        #[rust_name = "stroke_path"]
+        fn strokePath(self: Pin<&mut QPainter>, path: &QPainterPath, pen: &QPen);
```

Comment:
try to keep the ordering A->Z as it becomes increasingly hard to find methods :-)